tests/commands/test_deploy.py

```diff
@@ -97,3 +107,96 @@ def test_deploy(
         in caplog.text
     )
     assert f"🧾 Transaction hash: {hex(TX_HASH)}" in caplog.text
+
+
+@pytest.mark.parametrize(
+    "args, exp_class_hash, exp_salt, exp_abi",
+    [
+        (
+            [CONTRACT, 0, True, [], ALIAS, 0x424242, 5],  # args
+            "0x434343",
```

Comment:
maybe moving these to a `CLASS_HASH` constant (or array) for legibility? maybe also collapsing into multiple `parametrize()` statements, to get a better idea of the combinations

Reply:
I refactored for better readability (constants). Still, I don't like too much the idea of parametrizing the args because it would be adding seven more parameters at least to the `test_deploy_contract` method, which already has 9, and I don't think the benefit is big enough.